CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
-- [cygnus-ngsi][feature] Add truncation options based on time and size (#1317)
+- [cygnus-ngsi][feature] Add global data truncation options based on time and size (#1317)
```

Comment:
I've changed the title for the previous entry, accordingly to this new entry one.